python/tests/test_seldon_client.py

```diff
@@ -118,6 +125,32 @@ def test_predict_rest_json_data_seldon(mock_post):
     assert mock_post.call_count == 1
 
 
+@mock.patch("requests.post", side_effect=mocked_requests_post_success_raw_data)
+def test_predict_rest_raw_data_seldon_proto(mock_post):
+    sc = SeldonClient(
+        deployment_name="mymodel", gateway="seldon", client_return_type="proto"
+    )
+    response = sc.predict(raw_data=RAW_DATA_TEST)
+    json_response = seldon_message_to_json(response.response)
+    assert mock_post.call_args[1]["json"] == RAW_DATA_TEST
+    assert response.success is True
+    assert json_response == RAW_DATA_TEST
```

Comment:
Do we need assertions for the response values?

Reply:
acctually I kept the same correction checks as in the other test_predict_rest_ .... methods. 
clearly it can be remove - but why it is there for the first place?